boxoffice/models/item_collection.py

```diff
@@ -36,10 +36,10 @@ def url_for_view(self):
         }
     }
 
-    def roles_for(self, user=None, token=None):
-        if not user and not token:
+    def roles_for(self, actor=None, anchors=()):
+        if not actor and not anchors:
             return set()
```

Comment:
Remove this. Always call `super`, since you need the `all` and `auth` roles acquired from there.